src/devices/Hmc5883l/Status.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Hmc5883l
+{
+    /// <summary>
+    /// The status of HMC5883L device
+    /// </summary>
+    public enum Status
```

Comment:
make this `[Flags]` and match values with what the device returns

Reply:
Done